tests/heap.ipynb

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
please replace heap.ipynb to unittest module for heap

Reply:
I think this file should be removed and all the tests must be mentioned in the unit test file.